doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
2015-05-25 10:20 GMT+02:00 Raffi Enficiaud <notifications@github.com>:

> In doc/test_organization/managing_tests_dependencies.qbk
> <https://github.com/boostorg/test/pull/64#discussion_r30968211>:
>
> > @@ -7,17 +7,31 @@
> >
> >  [section:tests_dependencies Managing test dependencies]
> >
> > -Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators].
> > +In general, you should make sure that test cases are independent on one another.
>
> Yes. The link between test dependency and execution order is also not
> obvious.
>
I do not understand what you are saying with this. Being ordered before
some other test is being dependent on it, isn't it?